src/utils/BrambleExtensionLoader.js

```diff
@@ -9,9 +9,21 @@
 define(function (require, exports, module) {
     "use strict";
 
+    var ThemeManager = brackets.getModule("/view/ThemeManager");
+
     var basePath = PathUtils.directory(window.location.href);
 
     /**
+     * Themes must be loaded seperately from the regular extension loading
+     * process because we load from a server vs the filesystem. Add new
+     * themes that should be loaded by default here.
+     */
+    var brambleDefaultThemes = [
```

Comment:
Maybe just `brambleThemes`, since we don't have anything other than our defaults (i.e., it's implied).